pycolmap/pycolmap/estimators/generalized_absolute_pose.h

```diff
@@ -0,0 +1,95 @@
+#include "colmap/estimators/generalized_pose.h"
+#include "colmap/estimators/pose.h"
+#include "colmap/geometry/rigid3.h"
+#include "colmap/math/random.h"
+#include "colmap/optim/ransac.h"
+#include "colmap/scene/camera.h"
+
+#include "pycolmap/log_exceptions.h"
+#include "pycolmap/utils.h"
+
+#include <pybind11/eigen.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+using namespace colmap;
+using namespace pybind11::literals;
+namespace py = pybind11;
+
+py::object PyEstimateAndRefineGeneralizedAbsolutePose(
+    const std::vector<Eigen::Vector2d>& points2D,
+    const std::vector<Eigen::Vector3d>& points3D,
+    const std::vector<size_t>& camera_idxs,
+    const std::vector<Rigid3d>& cams_from_rig,
+    std::vector<Camera>& cameras,
+    const RANSACOptions& ransac_options,
+    const AbsolutePoseRefinementOptions& refinement_options,
+    const bool return_covariance) {
+  SetPRNGSeed(0);
```

Comment:
This may be something that we want to set outside and not hardcode to a fixed seed?

Reply:
I suggest to bind address this in a later PR: bind `SetPRNGSeed` and remove its calls in `pycolmap/estimators`

Reply:
Ok, wasn't aware this is not exposed through pycol map. Let's do it as a follow-up.